lang/po/Firmware_nl.po

```diff
@@ -0,0 +1,1787 @@
+# Translation of Prusa-Firmware into Dutch.
+#
+msgid ""
+msgstr ""
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Language: nl\n"
+"Project-Id-Version: Prusa-Firmware\n"
+"POT-Creation-Date: Thu, Oct 3, 2019 7:24:02 PM\n"
+"PO-Revision-Date: Thu, Oct 3, 2019 7:24:02 PM\n"
+"Language-Team: \n"
+"X-Generator: Poedit 2.0.7\n"
+"X-Poedit-SourceCharset: UTF-8\n"
+"Last-Translator: \n"
+"Plural-Forms: nplurals=3; plural=(n==1) ? 0 : (n>=2 && n<=4) ? 1 : 2;\n"
+
+# 
+#: 
+msgid "[%.7s]Live adj. Z\x0avalue set, continue\x0aor start from zero?\x0a%cContinue%cReset"
+msgstr "[%.7s]Z instell.\x0awaarde ingesteld,\x0averder of bij 0\x0abeginen?\x0a%cVerder%cReset"
```

Comment:
"terugvallen op 0" might be better than "bij 0 beginnen" for the intent of "revert to 0"